xMySql.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.1.0.0'
+ModuleVersion = '1.0.0.1'
```

Comment:
Revert this please, we'll iterate the version number when we publish to the master.